WordPress/Classes/ViewRelated/Notifications/Tools/IntrinsicTableView.swift

```diff
@@ -11,7 +11,14 @@ import Foundation
 ///
 class IntrinsicTableView: UITableView
 {
+    override var contentSize:CGSize {
+        didSet {
+            self.invalidateIntrinsicContentSize()
+        }
+    }
+
     override func intrinsicContentSize() -> CGSize {
+        layoutIfNeeded()
```

Comment:
Hmmm, not so sure about forcing layout within a calculated layout method. Might lead to issue down the road.